src/renderer.js

```diff
@@ -3,6 +3,7 @@ var cartodb = global.cartodb || {};
 var carto = global.carto || require('carto');
 var _ = global._ || require('underscore');
 var geo = require("./geo");
+var Filter = require("./filter");
```

Comment:
I don't understand why the Filter is part of the renderer, and why is renderer responsible of adding tiles to the Filter. I'd expect someone listening to the "tileAdded" event to load the data into an object where you can add filters.